internal/osutils/autostart/autostart_linux.go

```diff
@@ -133,9 +140,13 @@ func (a *app) disable() error {
 	if err != nil {
 		return errs.Wrap(err, "Could not find ~/.profile")
 	}
+	// Some older versions of the State Tool used a different ID for the autostart entry.
 	if fileutils.FileExists(profile) {
 		return sscommon.CleanRcFile(profile, sscommon.InstallID)
```

Comment:
I guess we're doing it this way in order to clean up the old ones? Can we clearly mark this as Legacy if so? ie. `InstallIDLegacy`.

Reply:
We still use `InstallID` for installations. So far it's only been using the `--` prefix in the `.profile` that has surfaced an issue.

Reply:
Oh interesting. In that case can we rename these constants so that it's clear they're related but for slightly different use-cases?

Reply:
I _believe_ this already is the case with the old one being `InstallID` and the new one being `AutostartID`. I'm open to any suggestions to make these more clear.